apps/andi/test/integration/andi_web/live/access_group_live_view/edit_access_group_live_view_test.exs

```diff
@@ -352,6 +352,51 @@ defmodule AndiWeb.AccessGroupLiveView.EditAccessGroupLiveViewTest do
     end
   end
 
+  describe "emits a user_access_group_associate event" do
```

Comment:
Would it beneficial to add a similar test for `disassociate` events?

Reply:
Yes! But that is part of the next card!